vitess.io/images/vitess_logo.svg

```diff
@@ -152,125 +152,125 @@
          d="m 367.1282,648.85881 -27.87054,-51.05696 43.91222,-28.31214 z"
```

Comment:
Can you please re-create the favicon as well?

It looks like I used Inkscape to open the .svg file and then export it to .png.

https://github.com/youtube/vitess/commit/3d4bf0d3c268728144cdf86db159fc62616203f1#diff-2901c8251c80036790228c936bc5a45a